### PR TITLE
[CDAP-13778] Disable deleting profile if its still enabled in profile action popover

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-13778
Build: https://builds.cask.co/browse/CDAP-UDUT70

### CODE REVIEW

**Review by elfenheart — DISMISSED** [review]
👍 LGTM

File: cdap-ui/app/cdap/components/Cloud/Profiles/ActionsPopover/ActionsPopover.scss [hunk 1]
```diff
@@ -41,6 +41,10 @@
       li.delete-action {
         color: $red-02;
       }
+      li.disabled {
+        cursor: not-allowed;
+        opacity: 0.6;
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/Cloud/Profiles/ActionsPopover/ActionsPopover.scss]: Seems like at most other places, we use `opacity: 0.5` for disabled state?

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 2]
```diff
@@ -239,6 +239,7 @@ features:
         deleteConfirmation: Are you sure you want to delete the profile *_{profile}_*?
         deleteError: There was a problem deleting the profile
         deleteTitle: Delete profile
+        disabledDeleteProfile: Profile needs to be enabled to delete
```
Comment (tonybach) [inline cdap-ui/app/cdap/text/text-en.yaml]: nit: this and the item below it seems out of order

**Comment by tonybach** [comment]
We are disabling `Delete` on profile list view if the profile is disabled, but we are not doing the same thing in profile detail view?

**Comment by ajainarayanan** [comment]
@tonybach No. Its the same popover component that is being used. So it should be disabled in both places.

**Comment by ajainarayanan** [comment]
@tonybach Fixed the check for disabling delete link.

**Review by tonybach — DISMISSED** [review]
LGTM 👍

**Review by tonybach — APPROVED** [review]
LGTM 👍

**Comment by leacd** [comment]
Reopened the jira and added comments.

**Pull request merged**